README.md

```diff
@@ -12,7 +12,7 @@ Add the following to your `build.gradle` file:
 ```groovy
 plugins {
     // Checker Framework pluggable type-checking
-    id 'org.checkerframework' version '0.6.38'
+    id 'org.checkerframework' version '0.6.37'
```

Comment:
@smillst this update looks like it's going in the wrong direction - I think it should be `0.6.39`